src/routes/Catalog/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Group } from '@mantine/core';
+import { useEffect } from 'react';
+import PageLoader from '../../components/PageLoader';
+import QuizCard from '../../components/QuizCard';
+import { useStore } from '../../useStore';
+
+export default function Catalog({ userName }: any) {
+  const { catalogList, getCatalogList } = useStore();
+
+  useEffect(() => {
+    getCatalogList();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  if (!catalogList) {
+    return <PageLoader />;
+  }
+
+  return (
+    <Group>
+      {catalogList.map((item, index) => (
+        <QuizCard
```

Comment:
Please combine QuizCard (Line No. 247) in Quizzes component with this
quizmaster/src/routes/Quizzes/index.tsx